source/shared/core_init.cpp

```diff
@@ -39,6 +39,8 @@ void core_sqlsrv_minit( _Outptr_ sqlsrv_context** henv_cp, _Inout_ sqlsrv_contex
     SQLSRV_STATIC_ASSERT( sizeof( sqlsrv_sqltype ) == sizeof( zend_long ) );
     SQLSRV_STATIC_ASSERT( sizeof( sqlsrv_phptype ) == sizeof( zend_long ));
 
+    setlocale(LC_ALL, "");
+
```

Comment:
Please add a comment here justifying this line 
Also, we only need this in non-Windows. Please use ifdef